R/gt.R

```diff
@@ -95,6 +95,16 @@ gt <- function(data,
     groupname_col <- NULL
   }
 
+  # Stop function if `rowname_col` and `groupname_col`
+  # have the same string values
+  if (!is.null(rowname_col) &&
+      !is.null(groupname_col) &&
+      any(rowname_col %in% groupname_col)) {
+
+    stop("Any values provided to `rowname_col` and `groupname_col` cannot be shared.",
```

Comment:
"The value `{rowname_col}` appears in both `rowname_col` and `groupname_col`; these arguments cannot have values in common."